clang/test/CodeGenSYCL/debug-info-file-prefix-map.cpp

```diff
@@ -0,0 +1,14 @@
+// RUN: %clang_cc1 -triple spir64-unknown-unknown -fsycl-is-device \
+// RUN: -fsycl-int-header=%t.header.h -fsycl-int-footer=%t.footer.h \
+// RUN: -main-file-name %S/Inputs/builtin.cpp -fsycl-use-main-file-name \
+// RUN: -full-main-file-name "%S/Inputs/builtin.cpp" \
+// RUN: -fmacro-prefix-map=%S/Inputs/= -fcoverage-prefix-map=%S/Inputs/= \
+// RUN: -fdebug-prefix-map=%S/Inputs/= \
+// RUN: -debug-info-kind=constructor -emit-llvm -O0 -o - "%S/Inputs/builtin.cpp" \
+// RUN: | FileCheck %s
```

Comment:
Please add a description for this test

Reply:
FullMainFileName is declared here [llvm/ llvm/CGDebugInfo.cpp at sycl · intel/llvm · GitHubCGDebugInfo.cpp at sycl · intel/llvm · GitHub](https://github.com/intel/llvm/blob/sycl/clang/lib/CodeGen/CGDebugInfo.cpp#L551) and represents the name of the original source file name.

Reply:
It's a codegen option triggered by https://github.com/intel/llvm/blob/sycl/clang/include/clang/Driver/Options.td#L6333

Reply:
Sorry if I wasn't clear earlier. I meant a description in this file. All test files need a comment describing what the test does.